### PR TITLE
Add object size and epoch lifetime

### PR DESCRIPTION
**Pull request by smallhive**

### CODE REVIEW

File: config/config.yaml [hunk 1]
```diff
@@ -24,8 +24,8 @@ neofs:
     passphrase: '' # Passphrase to decrypt wallet. If you're using a wallet without a password, place '' here.
     address:  NfgHwwTi3wHAS8aFAN243C5vGbkYDpqLHP # Account address. If omitted default one will be used.
   cid: 2qAEwyRwV1sMmq8pc32mKCt1SRmTBXrzP9KbfMoHmqYM
-  max_object_size: 209715200 # max object allowed to deploy via bearer token is 200mb.
   max_object_size: 209715200 # max object size allowed to be deployed via bearer token. 200mb.
+  max_object_lifetime: 345600 # max object lifetime. 4 days (in seconds).
```
Comment (roman-khimov) [inline config/config.yaml:28]: Can it be `time.Duration` like timeouts below? I'd like to specify `4d`, for example.

File: cmd/neofs-oauthz/config.go [hunk 2]
```diff
@@ -13,13 +13,14 @@ import (
...
-	defaultEmailAttr        = "Email"
-	defaultBearerCookieName = "Bearer"
-	defaultBearerLifetime   = 30
-	defaultMaxObjectSize    = 200 << 20 // 200MB
-	defaultConnectTimeout   = 30 * time.Second
-	defaultRebalanceTimer   = 15 * time.Second
-	defaultRequestTimeout   = 15 * time.Second
+	defaultEmailAttr         = "Email"
+	defaultBearerCookieName  = "Bearer"
+	defaultBearerLifetime    = 30
```
Comment (roman-khimov) [inline cmd/neofs-oauthz/config.go:18]: 30 of what? Epochs? Is it described anywhere?

File: config/config.yaml [hunk 3]
```diff
@@ -24,6 +24,8 @@ neofs:
     passphrase: '' # Passphrase to decrypt wallet. If you're using a wallet without a password, place '' here.
     address:  NfgHwwTi3wHAS8aFAN243C5vGbkYDpqLHP # Account address. If omitted default one will be used.
   cid: 2qAEwyRwV1sMmq8pc32mKCt1SRmTBXrzP9KbfMoHmqYM
+  max_object_size: 209715200 # max object allowed to deploy via bearer token is 200mb.
+  max_object_size: 209715200 # max object size allowed to be deployed via bearer token. 200mb.
```
[the remark this answers is not in the document]
Comment (smallhive) [reply config/config.yaml:27]: Rebase, rebase the rows duplicaze. Fixed

File: cmd/neofs-oauthz/config.go (hunk 2 above)
Comment (smallhive) [reply cmd/neofs-oauthz/config.go]: Epochs, added the comment for this

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**